bravado/fido_client.py

```diff
@@ -1,19 +1,15 @@
 # -*- coding: utf-8 -*-
 import logging
-
+import requests
 import six
 
 if six.PY3:
     raise ImportError("The fido client is not yet supported in py3")
 
 import fido
-from bravado_core.param import stringify_body as param_stringify_body
 from bravado_core.response import IncomingResponse
-from yelp_uri import urllib_utf8
```

Comment:
this is the only place where yelp_uri is used, you can remove the requirements from setup.py